### ISSUE TITLE
Feature request: delete droplet by tag

### ISSUE THREAD

**Issue by webignition** [toin0u/DigitalOceanV2#282]
I have the need to delete a droplet without necessarily knowing the id of the droplet.

The API [supports doing so](https://developers.digitalocean.com/documentation/v2/#deleting-droplets-by-tag) but neither the 4.3 or 4.4 branches appear to support this (apologies if I missed something).

The addition of a method to `DigitalOceanV2\Api\Droplet` seems straightforward and, aside from possible method name considerations, could take the form:

```php
public function removeAll(string $tag): void
{
    $this->delete('droplets', ['tag_name' => $tag]);
}
```

I'm happy to implement this change in a fork, including the addition of tests and the updating of documentation.

I'd just like to check:

 - that this change would be welcome (the functionality may be intentionally missing for reasons unknown to me)
 - which branch (4.3? 4.4?) to create a PR against
 - whether the method name `removeAll()` is acceptable

**Comment by webignition** [comment]
After attempting implementation, I realise that the above method suggestion won't work but have determined what will.

**Comment by GrahamCampbell** [comment]
Merged into 4.4. :)

**Comment by webignition** [comment]
Awesome!
